### PR TITLE
use lima to use melange on mac

### PR DESCRIPTION
**Pull request by developer-guy**
Signed-off-by: Batuhan Apaydın <batuhan.apaydin@trendyol.com>

/cc @kaniini @rawlingsj @dentrax

### CODE REVIEW

**Comment by rawlingsj** [comment]
Nice!  I'll probably defer to @kaniini and @jdolitsky as they probably know more about this

File: mac/README.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Currently `melange` relies on `apk`, which is currently
+not available for mac.
+
+This page documents workarounds to run
+`melange` on a mac.
+
+## OCI Container (Docker)
+
+Use the [official container image](https://github.com/chainguard-images/apko):
```
Comment (jdolitsky) [inline mac/README.md:11]: maybe better URL here would be https://github.com/chainguard-images/images/tree/main/images/melange

Comment (developer-guy) [reply mac/README.md]: Thank you, good catch @jdolitsky ☝️

**Comment by developer-guy** [comment]
any thoughts? @kaniini @imjasonh

**Comment by developer-guy** [comment]
kindly ping @kaniini

**Pull request merged**